unix/xserver/hw/vnc/vncInput.c

```diff
@@ -206,6 +206,9 @@ static int vncPointerProc(DeviceIntPtr pDevice, int onoff)
 		btn_labels[4] = XIGetKnownProperty(BTN_LABEL_PROP_BTN_WHEEL_DOWN);
 		btn_labels[5] = XIGetKnownProperty(BTN_LABEL_PROP_BTN_HWHEEL_LEFT);
 		btn_labels[6] = XIGetKnownProperty(BTN_LABEL_PROP_BTN_HWHEEL_RIGHT);
+		btn_labels[7] = XIGetKnownProperty(BTN_LABEL_PROP_BTN_BACK);
+		btn_labels[8] = XIGetKnownProperty(BTN_LABEL_PROP_BTN_FORWARD);
```

Comment:
Annoyingly enough, this is not what the evdev driver does. It marks these as "side" and "extra".

It's code is a bit annoying to decipher, but it matches what this code does:

https://gitlab.freedesktop.org/xorg/xserver/-/blob/master/hw/xfree86/drivers/inputtest/xf86-input-inputtest.c?ref_type=heads#L272

We should do the same for compatibility.